### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Abid-Chowdhury**

### CODE REVIEW

**Comment by Abid-Chowdhury** [comment]
Added a code of conduct

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+mr.abid2004@gmail.com.
```
Comment (SininAssets) [inline CODE_OF_CONDUCT.md:63]: My only comment would be, maybe like an exclusive enforcment email, but thats unecessary.

**Review by SininAssets — APPROVED** [review]
Code of conduct is Good

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Abid-Chowdhury) [reply CODE_OF_CONDUCT.md:63]: We would have to make another email account. I can add  / and also add your email if you like.

Comment (SininAssets) [reply CODE_OF_CONDUCT.md:63]: That can be added later, It was just a thought.

**Pull request merged**